docs/launch/local-db.md

```diff
@@ -1,37 +1,49 @@
 ---
 sidebar_label: Local DB
 sidebar_position: 1
-description: Managing Local Data with Dexie in a React Application
+description: Manage local data with Dexie in a web app built with the xmtp-js SDK
 ---
 
-# Managing Local Data with Dexie in a React Application
+# Manage local data with Dexie in a web app built with xmtp-js
 
-The performance of a web application can be significantly improved by leveraging local data storage. Particularly in the context of loading messages, using a local database can result in a 10x performance increase compared to solely relying on a network-based data source.
+<!--Ry - I believe we said that we'd make this tutorial about xmtp-js and the code is not about React.-->
 
-This guide provides a walkthrough on managing local data storage in a React application using the Dexie.js library. Dexie.js is a minimalistic wrapper for IndexedDB, which is a low-level API for client-side storage of significant amounts of structured data.
+<!--I'd also like to see if we can add this entire doc to the end of https://junk-range-possible-git-react-prod-rel-xmtp-labs.vercel.app/docs/build/local-first. WDYT?-->
+
+<!--Ry - In the title above, should we change "local data" to "local cache"? And change "local database" below to "local cache"? there are so many references to "database" and "database schema" in this doc - I don't know if they can all be changed to "cache" instead without changing the meaning. Perhaps this doc is really about database vs cache?-->
```

Comment:
they are basically aliases for one another. i think it makes sense to refer to the local-first architecture as having a "local cache," which is a database. and when referring to database specific terms/concepts, we can say database. WDYT?